src/translation-assistant.js

```diff
@@ -348,20 +361,42 @@ function createTemplate(englishStr, translatedStr, lang) {
 
 /**
  * Handles any per language special case translations, e.g. Portuguese uses
```

Comment:
Lets format the comment:

```
/**
 * Handles any per language special case translations, e.g. Portuguese uses
 * `sen` instead of `sin`.
 *
 * Many languages use decimal comma, according to this table:
 * https://docs.google.com/spreadsheets/d/1qgi-KjumcZ6yru19U5weqZK9TosRlTdLZqbXbABBJoQ/edit#gid=0
 *
 * TODO(danielhollas): For now only the obvious cases were implemented.
 * TODO(danielhollas): Need to update this when new langs join translations.
 *
 * @param {string} math A math expression to translate for locale.
 * @param {string} lang The locale of the translation language.
 * @returns {string} The translated math expression.
 */
```